src/app/modules/common/services.js

```diff
@@ -490,6 +518,12 @@ angular.module('BloomLibraryApp.services', ['restangular'])
 			return restangular.withConfig(authService.config()).one('classes/books', id).remove();
 		};
 
+            this.modifyBookField = function(book, field, value) {
+                var rBook = restangular.withConfig(authService.config()).one('classes/books', book.objectId);
+                rBook[field] = value;
+                rBook.put();
+            };
```

Comment:
I'm being lazy here, not looking for where you use this, but do we need to do anything to update any local copies of this book?